src/Components/Facility/Consultations/Feed.tsx

```diff
@@ -404,60 +405,89 @@ export const Feed: React.FC<IFeedProps> = ({ consultationId }) => {
+                  <div className="rounded-lg shadow-lg ring-1 ring-gray-400">
+                    <div className="rounded-t-lg bg-gray-100 px-6 py-4">
+                      <span className="block text-sm text-gray-800">
+                        Camera(s) preset
+                      </span>
+                    </div>
+                    <div className="flex flex-col">
+                      {bedPresets?.map((preset: any, index: number) => (
+                        <ButtonV2
+                          key={preset.id}
+                          variant={
+                            currentPreset === preset ? "primary" : "secondary"
+                          }
+                          onClick={() => {
+                            setLoading(CAMERA_STATES.MOVING.GENERIC);
+                            // gotoBedPreset(preset);
+                            absoluteMove(preset.meta.position, {
+                              onSuccess: () => {
+                                setLoading(CAMERA_STATES.IDLE);
+                                setCurrentPreset(preset);
+                                console.log(
+                                  "onSuccess: Set Preset to " +
+                                    preset?.meta?.preset_name
+                                );
+                                triggerGoal("Camera Preset Clicked", {
+                                  presetName: preset?.meta?.preset_name,
+                                  consultationId,
+                                  userId: authUser.id,
+                                  result: "success",
+                                });
+                              },
+                              onError: () => {
+                                setLoading(CAMERA_STATES.IDLE);
+                                setCurrentPreset(preset);
+                                console.log(
+                                  "onError: Set Preset to " +
+                                    preset?.meta?.preset_name
+                                );
+                                triggerGoal("Camera Preset Clicked", {
+                                  presetName: preset?.meta?.preset_name,
+                                  consultationId,
+                                  userId: authUser.id,
+                                  result: "error",
+                                });
+                              },
+                            });
+                            getCameraStatus({});
+                          }}
+                          className="w-full"
+                        >
+                          {preset.meta.preset_name || `Preset ${index + 1}`}
+                        </ButtonV2>
+                      ))}
+                    </div>
+                  </div>
+                </Popover.Panel>
+              </Transition>
+            </Popover>
```

Comment:
We already have a reusable component called `SelectMenuFormField`. Use that instead of duplicating the logic here...

Also, this does not look nice in desktop view...

<img width="433" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/1f572349-721d-4382-99ec-b9a57aa9e729">